src/app/helptext/network/interfaces/interfaces-list.ts

```diff
@@ -13,4 +13,10 @@ rollback_button: T("Discard"),
 rollback_changes_title: T("Discard Network Changes"),
 rollback_changes_warning: T("Discard unsaved network changes?"),
 changes_rolled_back: T("Network changes discarded."),
+
+checkin_title: T("Network Changes Check-in"),
+checkin_message: T("Network changes have been applied, check in now?  Settings will be automatically discarded if not checked in."),
```

Comment:
'Network settings have been changed. These changes will be lost if not saved. Save them now?'